RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1084,12 +1086,12 @@ boolean __restore(string resource_type, int goal, int meat_reserve, boolean useF
     }
   }
 
-  skill pick_blood_skill(){
-    int conservative_blood_bond_cost = hp_cost($skill[Blood Bond]) + ((9-hp_regen())*10);
+  skill pick_blood_skill(int final_hp){
```

Comment:
slightly confused about what `final_hp` is - I think it's "the minimum amount of HP that you must leave me with after using blood skills"? Comment would be great

Reply:
Looking up the chain, it is `my_hp()+o.vars["hp_total_restored"]`
that is being passed through 2 different functions to reach the current point
It comes from the line
`use_opportunity_blood_skills(o.vars["hp_restored_per_use"], my_hp()+o.vars["hp_total_restored"]);`